contractsV5/KyberNetwork.sol

```diff
@@ -489,9 +489,7 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
             bestReserve = reserveCandidates[0];
         }
 
-        bestDestAmount = destAmounts[bestReserve];
-    
-        return (reserveArr[bestReserve], bestDestAmount, isFeeLessReserve[reserveArr[bestReserve]]);
+        return (reserveArr[bestReserve], rates[bestReserve], isFeePayingReserve[address(reserveArr[bestReserve])]);
```

Comment:
so what is the logic
comparing dest amount but return exact rate.
sound good.

so all amount modifications are elsewhere.

Reply:
yes, that's correct.